app/components/channel_drawer/channels_list/filtered_list/index.js

```diff
@@ -25,15 +25,62 @@ const pastDirectMessages = createSelector(
     (directChannelsFromPreferences) => directChannelsFromPreferences.filter((d) => d.value === 'false').map((d) => d.name)
 );
 
+const getTeamProfiles = createSelector(
+    getProfilesInCurrentTeam,
+    (members) => {
+        return members.reduce((memberProfiles, member) => {
+            memberProfiles[member.id] = member;
+
+            return memberProfiles;
+        }, {});
+    }
+);
+
+function getGroupDetails(currentUserId, userIdsInChannels, profiles, groupChannels) {
+    return groupChannels.reduce((groupMemberDetails, channel) => {
+        if (!userIdsInChannels.hasOwnProperty(channel.id)) {
+            return groupMemberDetails;
+        }
+
+        const members = Array.from(userIdsInChannels[channel.id]).reduce((details, member) => {
+            if (member === currentUserId) {
+                return details;
+            }
+
+            const profile = profiles[member];
+            const username = `${details.username || ''} ${profile.username}`;
```

Comment:
@hmhealey what do you think about this? https://github.com/mattermost/mattermost-mobile/pull/755/commits/bb6e8ad2d1b7a675d1ef005d79c63f1b2fe1d737#diff-8e3ccfc258e37633e4c6bac473eedc6aR39